packages/diagrams/lib/src/cupertino_colors.dart

```diff
@@ -0,0 +1,688 @@
+                    ),
+                    ColorWidget(
+                      color: systemColor.values.first.highContrastColor,
+                      red: systemColor.values.first.highContrastColor.red,
+                      green: systemColor.values.first.highContrastColor.green,
+                      blue: systemColor.values.first.highContrastColor.blue,
+                    ),
+                    ColorWidget(
+                      color: systemColor.values.first.darkHighContrastColor,
+                      red: systemColor.values.first.darkHighContrastColor.red,
+                      green:
+                          systemColor.values.first.darkHighContrastColor.green,
+                      blue: systemColor.values.first.darkHighContrastColor.blue,
+                    ),
+                  ],
+                ),
+            ],
+          ),
+        );
+      case _label_colors:
+        return ConstrainedBox(
+          key: UniqueKey(),
+          constraints: BoxConstraints.tight(const Size(590.0, 830.0)),
+          child: Table(
+            columnWidths: const <int, TableColumnWidth>{
+              0: FixedColumnWidth(150),
+              1: FixedColumnWidth(110),
+              2: FixedColumnWidth(110),
+              3: FixedColumnWidth(110),
+              4: FixedColumnWidth(110),
+            },
+            defaultVerticalAlignment: TableCellVerticalAlignment.middle,
+            children: <TableRow>[
+              TableRow(
+                children: <Widget>[
+                  Text('Name', style: textStyle, textAlign: TextAlign.center),
+                  Text('Color', style: textStyle, textAlign: TextAlign.center),
+                  Text('Dark color',
+                      style: textStyle, textAlign: TextAlign.center),
+                  Padding(
+                    padding: const EdgeInsets.only(bottom: 14.0),
+                    child: Text('High\nconstrast color',
+                        style: textStyle, textAlign: TextAlign.center),
+                  ),
+                  Padding(
+                    padding: const EdgeInsets.only(bottom: 14.0),
+                    child: Text('Dark high\ncontrast color',
+                        style: textStyle, textAlign: TextAlign.center),
+                  ),
+                ],
+              ),
+              for (Map<String, CupertinoDynamicColor> labelColor in labelColors)
+                TableRow(
+                  children: <Widget>[
+                    Text(
+                      labelColor.keys.first,
+                      style: textStyle,
+                      textAlign: TextAlign.center,
+                    ),
+                    ColorWidget(
+                      color: labelColor.values.first.color,
+                      red: labelColor.values.first.red,
+                      green: labelColor.values.first.green,
+                      blue: labelColor.values.first.blue,
```

Comment:
simplify by just passing `labelColor.values.first` and have `ColorWidget` get the four arguments

Reply:
Great idea!